Makefile

```
@@ -0,0 +1,78 @@
+# 2. Start Claude Code (new session)
+claude: up
+	@echo "🤖 Starting Claude Code (new session)..."
+	@devcontainer exec --workspace-folder . -- \
+		claude --dangerously-skip-permissions --verbose $(ARGS)
+
+# 3. Continue previous session
+continue: ARGS=-c
+continue: claude
+
+# 4. Pause the running devcontainer
+pause:
```

checkmake (0.2.2)

[warning] Target body for "pause" exceeds allowed length of 5 (8).

(maxbodylength)